core/services/ocr2/database.go

```diff
@@ -299,7 +322,7 @@ WHERE ocr2_oracle_spec_id = $1 AND config_digest = $2
 }
 
 func (d *db) DeletePendingTransmission(ctx context.Context, t ocrtypes.ReportTimestamp) (err error) {
-	_, err = d.ExecContext(ctx, `
+	_, err = d.q.WithOpts(pg.WithLongQueryTimeout()).ExecContext(ctx, `
```

Comment:
Why do you think this might run long? I would think it's O(1) if indexed.
`DeletePendingTransmissionsOlderThan` seems more open ended, but should be quick and indexed too.

Reply:
I added the timeout here because on logs we received this method timed out, so I'm trying to work around that.